tests/integration/models/os.spec.coffee

```diff
@@ -5,7 +5,7 @@ bSemver = require('balena-semver')
 
 { balena, givenLoggedInUser, IS_BROWSER } = require('../setup')
 
-{ osVersionRCompare } = require('../../../build/util')
+{ osVersionRCompare } = require('../../osVersionRCompare')
```

Comment:
It seems this is not being used anywhere.

Reply:
But it is, on line 112
See: https://github.com/balena-io/balena-sdk/pull/683/files/c9a5b16f16e939fda96dd599dc308d3190bae8ab#diff-4d287dbe36ded3f6fd75fa6cc8ff9b36R112

Reply:
Ok, then my searching sucks, I think I searched for `osVersionCompare`.